src/libs/prefs.js

```diff
@@ -1,5 +1,5 @@
-import { getUser } from 'src/libs/auth'
 import { getDynamic, getLocalStorage, setDynamic } from 'src/libs/browser-storage'
+import { getUser } from 'src/libs/state'
```

Comment:
libs/auth re-exports getUser from libs/state.

https://github.com/DataBiosphere/terra-ui/blob/979645fe9d10352dd16eb5a1d920f92069da57b7/src/libs/auth.js#L21

Changing this import resolves a circular import (auth imports notifications imports prefs imports auth) that was causing problems in tests.